README.md

```diff
@@ -1,5 +1,19 @@
 # Cb Response Event Forwarder
 
+## NEW FOR 4.0
```

Comment:
Love the changelog - move this down to the `# Changelog` section below. We will expand on this in a blog post announcing the changes.